js/src/lib/components/Icons/IconDepthEstimation.svelte

```diff
@@ -0,0 +1 @@
+<svg width="1em" height="1em" viewBox="0 0 32 32"><path d="M29.474 19.12L23.681 16l5.793-3.12a1 1 0 0 0 0-1.76l-13-7a.998.998 0 0 0-.948 0l-13 7a1 1 0 0 0 0 1.76L8.319 16l-5.793 3.12a1 1 0 0 0 0 1.76l13 7a1 1 0 0 0 .948 0l13-7a1 1 0 0 0 0-1.76zM16 6.136L26.89 12L16 17.864L5.11 12zm0 19.728L5.11 20l5.319-2.864l5.097 2.744a1 1 0 0 0 .948 0l5.097-2.745L26.891 20z" fill="#888888"></path></svg>
```

Comment:
two points:
1. Could you add the script tag & class={classNames} as in other icons. Example here https://github.com/huggingface/hub-docs/blob/12c9de46be6a3d9bbb54658040144eb028ebd593/js/src/lib/components/Icons/IconCaretDown.svelte#L1-L6
2. Run `npm run format` which will format this file

Otherwise, looks great to me !

Reply:
Thanks, @mishig25! Just pushed the changes. LMKWYT.